FirebaseMessaging/Sources/Token/FIRMessagingTokenManager.m

```diff
@@ -458,41 +457,34 @@ - (void)deleteWithHandler:(void (^)(NSError *))handler {
  *  since the Keychain items are marked with `*BackupThisDeviceOnly`.
  */
 - (void)resetCredentialsIfNeeded {
-  BOOL checkinPlistExists = [_checkinStore hasCheckinPlist];
+  BOOL checkinPlistExists = [_authService hasCheckinPlist];
   // Checkin info existed in backup excluded plist. Should not be a fresh install.
   if (checkinPlistExists) {
     return;
   }
-
-  // Resets checkin in keychain if a fresh install.
   // Keychain can still exist even if app is uninstalled.
-  FIRMessagingCheckinPreferences *oldCheckinPreferences = [_checkinStore cachedCheckinPreferences];
-
-  if (oldCheckinPreferences) {
-    [_checkinStore removeCheckinPreferencesWithHandler:^(NSError *error) {
-      if (!error) {
-        FIRMessagingLoggerDebug(
-            kFIRMessagingMessageCodeStore002,
-            @"Removed cached checkin preferences from Keychain because this is a fresh install.");
-      } else {
-        FIRMessagingLoggerError(
-            kFIRMessagingMessageCodeStore003,
-            @"Couldn't remove cached checkin preferences for a fresh install. Error: %@", error);
-      }
-      if (oldCheckinPreferences.deviceID.length && oldCheckinPreferences.secretToken.length) {
-        FIRMessagingLoggerDebug(kFIRMessagingMessageCodeStore006,
-                                @"App reset detected. Will delete server registrations.");
-        // We don't really need to delete old FCM tokens created via IID auth tokens since
-        // those tokens are already hashed by APNS token as the has so creating a new
-        // token should automatically delete the old-token.
-        [self didDeleteFCMScopedTokensForCheckin:oldCheckinPreferences];
-      } else {
-        FIRMessagingLoggerDebug(kFIRMessagingMessageCodeStore009,
```

Comment:
Is this message still relevant?

Reply:
FYI since the message is still relevant: The conditional block where it's logged is missing in the latest commit.

Reply:
You are right, it's still relevant. I'm adding it back.